### PR TITLE
experimental: Commands UI design tweaks

### PR DESCRIPTION
**Pull request by kof**
## Description

- Bigger border radius for the dialog and floating panels as well for consistency
- Bigger search container height, same as titles in dialogs

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 0000)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: packages/design-system/src/components/command.tsx [hunk 1]
```diff
@@ -30,23 +30,23 @@ import { Button } from "./button";
...
   border: `1px solid ${theme.colors.borderMain}`,
+  borderRadius: theme.borderRadius[7],
   // clip selected item background
   overflow: "clip",
   // remove input border bottom when no command matches
-  [inputBorderBottomSize]: "0px",
+  [inputBorderBottomSize]: 0,
   "&:has([cmdk-group]:not([hidden]))": {
-    [inputBorderBottomSize]: "1px",
+    [inputBorderBottomSize]: 1,
```
Comment (TrySound) [inline packages/design-system/src/components/command.tsx:49]: You sure `1 solid red` will work?

File: packages/design-system/src/components/command.tsx [hunk 2]
```diff
@@ -30,23 +30,23 @@ import { Button } from "./button";
 import { Popover, PopoverContent, PopoverTrigger } from "./popover";
 import { Kbd } from "./kbd";
 
-const panelWidth = "400px";
-const itemHeight = "32px";
+const panelWidth = 400;
+const itemHeight = 32;
```
Comment (TrySound) [inline packages/design-system/src/components/command.tsx:34]: please let's not rely on stitches magic where it is not colocated

File: packages/design-system/src/components/command.tsx [hunk 3]
```diff
@@ -101,8 +101,8 @@ export const Command = (props: CommandProps) => {
 
 const CommandDialogContent = styled(DialogContent, {
   position: "absolute",
-  top: 140,
-  left: `calc(50% - ${panelWidth} / 2)`,
```
Comment (TrySound) [inline packages/design-system/src/components/command.tsx:105]: there is nothing wrong with having it in calc

**Review by johnsicili — APPROVED** [review]
Good. Drag handles are a little to close to the edge imo but very minor

Screenshot 2024-12-16 at 7 01 17 AM

File: packages/design-system/src/components/command.tsx (hunk 1 above)
Comment (kof) [reply packages/design-system/src/components/command.tsx:49]: no, but we compile it to 1px, so we can use numbers, right?

File: packages/design-system/src/components/command.tsx (hunk 2 above)
Comment (kof) [reply packages/design-system/src/components/command.tsx:34]: hmm ok

File: packages/design-system/src/components/command.tsx (hunk 1 above)
Comment (TrySound) [reply packages/design-system/src/components/command.tsx:49]: who compiles it? Stitches magic? I have no idea if it compiles custom properties or not. Too magic.

**Pull request merged**